tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml

```
@@ -308,6 +308,13 @@ spec:
       command: [ec]
       args:
         - version
+    
+    - name: show-config
+      image: quay.io/enterprise-contract/ec-cli:snapshot
+      command: [jq]
+      args:
+        - ".policy.sources[0].config"
```

Oh I think it should be just this:

```suggestion
        - ".policy"
```

(I guess the name is confusing since we call different things "policy" and "config" in different contexts.)

---

jq .policy would show all the content of policy, not just config.